<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Hepadnavirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
    
      <h1 class="project-name">Hepadnavirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of hepadnaviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./news.html" class="btn">News</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/tree/master" class="btn">View on GitHub</a>

    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous hepadnavirus (eHBV) data</strong>
	   </h3>


		 <p>

		   Whole genome sequencing has revealed the presence of DNA sequences derived from
		   hepadnaviruses in vertebrate genomes. These <b>‘endogenous hepatitis B viruses’
		   (eHBVs)</b> are thought to have originated via ‘germline incorporation’ events in
		   which hepadnavirus DNA sequences were integrated into chromosomal DNA of germline
		   cells and subsequently inherited as novel host alleles. 
		   eHBV sequences are in some ways
		   equivalent to hepadnavirus ‘fossils’ in that they provide a source of retrospective
		   information about the distant ancestors of modern hepadnaviruses. 
		 </p>


		 <p>
		   We created an <b><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE">extension</a></b> to Hepadnavirus-GLUE to organise the 
		   '<b>genomic fossil record</b>' of hepadnaviruses.
		   This page provides a description of this extension, and
		   quick links to specific data items.
		</p>
	   

		<p>
		  <b>Please note</b>: links to files on GitHub are mainly designed to indicate where
		  these files are located within the repository. 
		  To investigate files (e.g. tree files) in the appropriate software context we recommend
		  <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/zipball/master"><b>downloading the entire repository</b></a>
		  and browsing locally. 	  
		</p>



	   <br>
         <p><img src="./../assets/images/hepadnavirus-header.jpg" alt="Timeline of hepadnavirus evolution" /></p>



<blockquote>
           <b>Timeline of hepadnavirus evolution</b>. 
		   The inset panel shows the evolutionary relationships of hepadnavirus genera,
		   the larger tree is a time-calibrated evolutionary tree of vertebrates.
		   Geological eras are indicated by background shading. The scale bar shows
		   time in millions of years before present.           
		   For details see <a target="_blank" href="https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veab012/6157734">Lytras <i>et al</i></a> (2020).	
		   Mya=Million years ago. 
</blockquote>
		 



	   <h3>
		 <a id="Virology" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to molecular biological studies of hepadnaviruses</strong>
	   </h3>        
	   <hr>

	   <p>
          Endogenous viral sequences can inform our understanding of contemporary viruses
          in a wide variety of ways. Perhaps most importantly, EVEs allow calibration of
          the long-term evolutionary history of virus groups, which greatly influences 
          how we understand their biology.
 	   </p>

	   <p>
         Importantly, once time calibrations have been established, a far richer range of
         comparative genomic studies can be performed. By examining
         variation in the light of a known evolutionary history, these studies can provide 
         invaluable insights into the biological mechanisms through which viruses 
         replicate and spread.
	   </p>



        <p><img src="./../assets/images/hepadna-paleovirus-header.jpg" alt="Hepadnavirus EVEs" /></p>


<blockquote>
		<p>
		  Some of the species in which we have identified <b>EVEs derived from hepadnaviruses</b> 
		  (commonly referred to as <b>endogenous hepatitis B (eHBV) elements</b>)
		  <br>
		  From left to right:
		  emperor penguin (<i>Aptenodytes forsteri</i>), barn owl (<i>Tyto alba</i>), cormorants (family Phalacrocoracidae), Anna's hummingbird (<i>Calypte anna</i>).
		</p>
         
</blockquote>
		 


     
	   <br>
	   <h3>
		 <a id="Metagenomics" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to viral metagenomics</strong>
	   </h3>        
	   <hr>

	   <p>
          The eHBV sequences in Hepadnavirus-GLUE can
          provide a useful resource for those
          interested in identifying and characterising hepadnaviruses in metagenomic
          datasets.
	   </p>


	   <p>
          Firstly, the eHBV sequences collated here can be used to exclude any potential
          'false positive' hits (i.e. sequences that seem to represent new hepadnaviruses but in fact derive
          from genomic DNA).
	   </p>


	   <p>
          In addition, when new hepadnavirus species are identified, inclusion of EVEs
          in phylogenetic analyses can often provide useful information
          about their broader ecology and evolution, including (uniquely) their long-term
          evolution. 
	   </p>
       

       
	   <br>
	   <h3>
		 <a id="Genomics" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to genomics</strong>
	   </h3>        
	   <hr>

	   <p>
          eHBVs are not only useful genetic markers, several lines of evidence indicate
          they may have, or have had, functional roles as host alleles. The prevalence of 
          multicopy eHBV lineages in some species suggests that germline incorporation of
          hepadnavirus sequences might have influenced the evolution of host genomes in
          important ways.
	   </p>

	   <p>

          Consistent with the idea that germline incorporation of hepadnavirus
          sequences might in some cases be favoured by selection at the level of the host,
          we have identified several examples of loci containing multiple fixed eHBV elements
          (<a target="_blank" href="https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veab012/6157734">Lytras <i>et al</i>, 2020</a>),
          each derived from a distinct germline colonisation event. It remains unclear
          whether this reflects natural selection due to a favourable influence or
          preferential integration of hepadnaviruses into particular loci (e.g. because they
          are accessible in embryonic cells).      

	   </p>


        <p><img src="./../assets/images/hepadna-paleovirus-midlevel.jpg" alt="Hepadnavirus EVEs" /></p>


<blockquote>
		<p>
		  Additional species in which we have identified <b>endogenous hepatitis B (eHBV) elements</b>
		  <br>
		  From left to right:
		  tuatara (<i>Sphenodon punctatus</i>), ducks (family Anatidae), snakes (suborder Serpentes), red-legged seriema (<i>Cariama cristata</i>).
		</p>
         
</blockquote>
		 



 
 	   <br>
	   <h3>
		 <a id="EveProject" class="anchor" href="#EveProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>The EVE component of Hepadnaviridae-GLUE</strong>
	   </h3>
	   <hr>

	   
		<p>

          Currently, the distribution and diversity of hepadnavirus-related sequences
          in animal genomes remains incompletely characterized.
          Progress in characterising these elements has been hampered by the challenges
          encountered attempting to analyse their fragmentary and degenerated sequences. 
          <b>Hepadnavirus-GLUE</b> aims to address these issues.
		</p>

	   
		<p>
          We have incorporated into this project a set of principles for
          organising the hepadnavirus 'fossil record', and a protocol 
          through which it can be accessed and collaboratively developed.
		</p>

        <p>
          This website provides background information about the EVE component of 
          <b>Hepadnavirus-GLUE</b>, and direct links to specific data items. 

		</p>

 	   <br>
	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for eHBVs</strong>
	   </h3>
	   <hr>

		<p>
          We have applied a systematic approach to naming eHBV, following a convention 
          developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>


 
        <p><img src="./../assets/images/nomenclature-eHBV.jpg" alt="eHBV Nomenclature" /></p>

 		


		<p>
          The first component is the classifier ‘eHBV’ (endogenous hepatitis B virus/endogenous hepadnavirus). 
		</p>

		<p>
          The second component is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of eHBV group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer that identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
 		</p>
		
		<p>
          Where an EVE sequence is thought to have been duplicated within the germline
          following it's initial incorporation (e.g. via segmental duplication or transposition)
          we have appended an additional 'duplicate id' to the numeric ID, separated by a period.
          Please note that we have not yet resolved the orthologous relationships among
          sets of eHBV sequences belonging to multicopy eHBV lineages. We
          have therefore assigned unique duplicate IDs to each sequence within these lineages. 
		</p>
		
		<p>
          The third component of the ID defines the set of host species in which
          the ortholog occurs, or did occur prior to being deleted.
  		</p>



	   <h3>
		 <a id="Avi.1.Neoaves" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Highlighted EBVs: Avi.1-Neoaves</strong>
	   </h3>
	   <hr>



		<p>
          eHBV-Avi.1-Neoaves was the first hepadnavirus-derived EVE to be described

          (<a target="_blank" href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191">Katzourakis and Gifford, 2010</a>;
          <a target="_blank" href="https://www.nature.com/articles/ncomms2798">Suh et al, 2013</a>).
          It is located in an intron of the
          <a target="_blank" href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=FRY"><b>FRY Microtubule Binding Protein</b></a></li>
       
          gene and derives from a germline incorporation
          event involving an ancient <b>avihepadnavirus</b>.
          This event predates the divergence of the avian clade <b>Neoaves</b>, which includes 
          all modern birds except the Galloanserae (ducks and chickens) and Palaeognathae.
          Consequently, an Avi.1-Neoaves insertions are present in multiple avian species.
 		</p>




        <p><img src="./../assets/images/ehbv-avi.1-neoaves.jpg" alt="Background images for Avi.28-Alauda" /></p>
 		


<blockquote>
		  Left panel: the clade 'Neoaves' includes almost all modern birds, including (clockwise)
		  flamingos (order Phoenicopteriformes), parrots (order Psittaciformes),
		  songbirds (order Passeriformes) and all birds of prey.
		  Right panel: Analysis using Hepadnavirus-GLUE has revealed that at least four distinct
          clades (I-IV) of avihepadnavirus have existed during hepadnavirus evolution.

</blockquote>


		<p>
		  Links to related data items: 

		  <ul>
			<li> <a target="_blank" href="../../sources/fasta-refseqs-ehbv/ehbv-avi.1-neoaves.fna">eHBV-Avi.1-Neoaves nucleotide consensus sequences</a></li>
		    <li> <a target="_blank" href="./../assets/images/avihepadna-phylogeny.jpg">Avihepadnaviridae phylogeny with EVEs</a></li>
		    <li> <a target="_blank" href="./../assets/images/avian-phylogeny-part1.jpg">Avian phylogeny (part 1)</a> (taken from <a target="_blank" href="https://dx.doi.org/10.1038/nature15697">Prun <i>et al</i></a>)</li>
		    <li> <a target="_blank" href="./../assets/images/avian-phylogeny-part2.jpg">Avian phylogeny (part 2)</a> (taken from <a target="_blank" href="https://dx.doi.org/10.1038/nature15697">Prun <i>et al</i></a>)</li>
		  </ul>

  		</p>



 	   <br>
	   <h3>
		 <a id="Avi.14-Gavia-Avi.15-Gavia" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Highlighted EBVs: Avi.14-Gavia and Avi.15-Gavia</strong>
	   </h3>
	   <hr>

		<p>
          These two eHBV loci in the genome of the common loon (<i>Gavia immer</i>) 
          derive from germline incorporation events involving
          ancient avihepadnaviruses.
          The viruses involved belong to a distinct subgroup of avihepadnaviruses, which we
          labeled 'clade II'. So far this group of hepadnaviruses is only known from eHBV
          sequences, and it remains unknown whether any modern representatives still
          circulate in birds.
		</p>

        <p><img src="./../assets/images/ehbv-avi.14+15-gavia.jpg" alt="Background images for Avi.14-Gavia and Avi.15-Gavia" /></p>
 		
 		


<blockquote>
		<p>
		  Left to right: (i) The common loon (<i>Gavia immer</i>) contains two eHBV insertions derived from clade II aviheapdnaviruses.
		  (ii) Emperor penguins also contain clade II avihepadnaviruses, indicating their biogeographic host range could extend to the Antarctic;    
		  eHBV insertions derived from clade II were also identified in (iii) Great crested grebe (<i>Podiceps cristatus</i>) and; (iv) barn owl (<i>Tyto alba</i>) 
		</p>        
</blockquote>
	
 

		<p>
          The two insertions are similar in that they both span near-complete hepadnaviral
          genomes. Remarkably, however, they appear to have been acquired in entirely
          different germline integration events.
          
          In our recent paper we show that - consistent with this - both insertions show
          a greater degree of divergence in a region of the Surface protein associated
          with binding to host receptors than they do in other genomic regions.
 		</p>



	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="../../sources/fasta-refseqs-ehbv/ehbv-avi.14-gavia.fna">Avi.14-Gavia nucleotide sequence</a></li>
		   <li> <a target="_blank" href="../../sources/fasta-refseqs-ehbv/ehbv-avi.15-gavia.fna">Avi.15-Gavia nucleotide sequence</a></li>
		   <li> <a target="_blank" href="./../assets/images/cladeII_var_fn.png">Similarity plots comparing clade II eHBV sequences</a></li>
		 </ul>

	   </p>
 
 	   <br>
	   <h3>
		 <a id="ehbv-avi.27-Phalacrocoracidae" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Highlighted EBVs: Avi.27-Phalacrocoracidae</strong>
	   </h3>
	   <hr>



		<p>
          Avi.27-Phalacrocoracidae is a <b>multicopy</b> eHBV lineage identified in <b>cormorants</b>
          (family Phalacrocoracidae).
          We identified approximately 300 copies of this avihepadnavirus derived element
          in the cormorant germline.
          All avi.27 elements are quite short, spanning around 350 nucleotides of the polymerase gene.
          Analysis of flanking DNA regions suggested that the insertion has been mobilised
          within the cormorant germline by <b>transposable elements (TEs)</b>.

 		</p>


        <p><img src="./../assets/images/ehbv-avi.27-phalacrocoracidae.jpg" alt="Background images for Avi.28-Alauda" /></p>
 		



<blockquote>

		<p>
          Cormorants have many anatomical and physiological adaptations for diving,
          including amphibious hearing.
		  Left to right: (i) and (ii) Great cormorant (<i>Phalacrocorax carbo</i>);
		  (iii) Brandt's cormorant  (<i>Phalacrocorax penicillatus</i>); 
		  (iv) Cormorant fishing is a traditional fishing method in which fishermen use
		  trained cormorants to fish in rivers. Historically, cormorant fishing has taken
		  place in Japan and China, as well as some European countries.
		</p>
         
</blockquote>

		<p>
  
          Phylogenies indicate that the initial germline incorporation event that gave rise
          to this multicopy eHBV lineage predates the diversification of cormorants.
          How did so many distinct eHBV/TE insertions get fixed in the cormorant germline?
          Possibly, the capture of EVE sequences offered a selective advantage to TE lineages
          that allowed them to proliferate so widely that numerous copies were fixed only by chance.
          Alternatively, TE sequences containing hepadnavirus-derived DNA might,
          for some reason, be more likely to be fixed. 
 
 		</p>

 
		<p>
          Numerous lines of evidence indicate that TEs play a role in 
          <a target="_blank" href=" https://doi.org/10.1371/journal.pgen.1003234"><b>rewiring gene regulatory networks</b></a>.
          This process is thought to underlie many of the evolutionary
          adaptations seen in vertebrates. Possibly, fixation of eHBV-avi.28 copies
          in cormorants genomes reflects a role for these eHBVs - or the TEs
          they are embedded within - in facilitating cormorant evolution and adaptation.
 		</p>



	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/sources/fasta-refseqs-ehbv/ehbv-avi.27-phalacrocoracidae.fna">Avi.27-Phalacrocoracidae consensus sequence</a></li>
		   <li> <a target="_blank" href="./../assets/images/avihepadna-phylogeny.jpg">Avihepadnaviridae phylogeny with EVEs</a></li>
		   <li> <a target="_blank" href="./../assets/images/avi.27-flanks-analysis.png">Analysis of genomic regions flanking avi.27 insertions</a></li>
		 </ul>

	   </p>
		

 	   <br>
	   <h3>
		 <a id="'Avi.28-Alauda" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Highlighted EBVs: Avi.28-Alauda</strong>
	   </h3>
	   <hr>


        <p><img src="./../assets/images/ehbv-avi.28-alauda.jpg" alt="Background images for Avi.28-Alauda" /></p>
 		

<blockquote>
		Eurasian skylark (<i>Alauda arvensis</i>).
		Skylarks sing beautiful, complicated songs over grasslands, rolling downs, coastal saltmarshes,
		and wide arable fields. 
</blockquote>

		<p>
		  We identified a full-length eHBV insertion in the genome of the 
		  <b>Eurasian skylark</b> (<i>Alauda arvensis</i>).
		  This sequence is notable for its close relationship to avihepadnavirus
		  species that currently circulate in birds.
          All extant avihepadnaviruses fall into one subgroup (which we labeled 'IV' in our study). 
          Avi.28.Alauda groups as a basal member of this clade.
 		</p>

	
  	
	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="../../sources/fasta-refseqs-ehbv/ehbv-avi.28-alauda.fna">eHBV-Avi.28-Alauda nucleotide sequence</a></li>
		   <li> <a target="_blank" href="./../assets/images/avihepadna-phylogeny.jpg">Avihepadnaviridae phylogeny with EVEs</a></li>
		 </ul>

	   </p>



	   <h3>
		 <a id="'Meta.1.Sauria" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Highlighted EBVs: Meta.1-Sauria</strong>
	   </h3>
	   <hr>


		<p>
		  Meta.1.Sauria is - we think - the oldest EVE ever described. We identified
		  putatively orthologous copies in a lizard - the tuatara (<i>Sphenodon</i>) -
		  as well as in multiple avian species. This implies that germline integration 
		  occurred prior to the divergence of these lineages in the late <b>Paleozoic Era</b> 
		  (~250-300 million years ago).
   		</p>


		<p>
 		  This insertion also provides important new insights into the evolution of
		  hepadnaviruses.
          It belongs to the recently described <b><i>Metahepadnavirus</i></b> genus.
		  So far, metahepadnaviruses have only been identifiied in fish.
		  Unexpectedly, these fish viruses were found to group relatively closely with
		  mammalian hepadnaviruses (genus <i>Orthohepadnavirus</i>). This seemed to imply 
		  a transfer of viruses between fish and mammals. However, eHBV sequences reveal
		  that metahepadnaviruses have an ancient origin and - historically at least -
		  a broad distribution. 
   		</p>



        <p><img src="./../assets/images/ehbv-meta.1-sauria.jpg" alt="Species containing Meta.1-Sauria insertions in their genomes" /></p>
 		

<blockquote>
		  Metahepadnaviruses are a recently discovered group of hepadnaviruses left to right:
		  (i) White sucker ( ) - one of the fish species in which metahepadnaviruses were first identfied;
		  (i) eHBVs derived from metahapadnaviruses were identified in the tuatara (<i>Sphenodon punctatus</i>) - a highly divergent reptile;
		  (iii) Tuataras are the only surviving representatives of the ancient reptile lineage that contained 
		  the marine ichthyosaurs and pleisiosaurs;
		  (iv) Meta.1-Sauria insertions have also been preserved in some avian genomes.
		  Emperor penguins contain a relatively well-preserved ortholog.
		  
</blockquote>
	



		<p>
		  Links to related data items: 
		  <ul>
		  
		    <li> <a target="_blank" href="../../sources/fasta-refseqs-ehbv/ehbv-meta.1-neoaves.fna">Avian consensus</a></li>
			<li> <a target="_blank" href="../../sources/fasta-digs-ehbv/ehbv-meta.1-Sphenodon_punctatus.fas">Sphenodon ortholog</a></li>
			<li> <a target="_blank" href="./../assets/images/meta.1-sauria-orthology.png">Evidence for <i>Sphenodon</i>/Aves orthology</a> </li>
			
		  </ul>

  		</p>
		





 
	   <br>
	   <h3>
		 <a id="Raw eHBV sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw eHBV sequences and data</strong>
	   </h3>
	   <hr>

		<p>
          These are the raw data generated by
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>.
          The 
          <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/tabular/eve/ehbv-side-data.tsv"><strong>tabular file</strong></a>
          contains information about the genomic location of each EVE. 
          EVEs were classified by comparison to a
          <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/digs/hepadna_refs_consolidate_final.faa">reference library of polypeptide sequences</a>
          designed to represent the known diversity of hepadnaviruses - this includes
          extinct lineages represented only by endogenous viral elements (EVEs).
 		</p>

		<p>
          These data were obtained via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI genomes</a>
          (2020-07-15).
		</p>


		<p>
           Raw data about the EVEs in tabular format can be found <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/tabular/eve/ehbv-side-data.tsv"><strong>here</strong></a>.
  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files) can be found <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/tree/master/sources/fasta-digs-ehbv"><strong>here</strong></a>.
  		</p>
		

    
         
	   <br>
	   <h3>
		 <a id="eHBV reference sequences and data" class="anchor" href="#Ref_eHBV_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>eHBV reference sequences and data</strong>
	   </h3>
	   <hr>

		<p>
           We constructed consensus sequences for paleoviruses by aligning eHBV
           sequences derived from the same initial germline colonisation event - i.e.
           orthologs in distinct species, and paralogs that have arisen via intragenomic
           duplication.
  		</p>

		<p>
           Reference sequence data in tabular format are <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/tabular/eve/ehbv-refseqs-side-data.tsv"><strong>here</strong></a>.
  		</p>
	   <br>

		<p>
           <strong>Nucleotide level data:</strong>
  		</p>
		
		<p>

		<table>
		
			<thead>
				<tr>
					<th>Taxonomic group</th>
					<th>Full-length eHBV</th>
					<th>Core codons</th>
					<th>Surface codons</th>
					<th>Pol codons</th>
				</tr>
			</thead>
		
			<tbody>

				<tr>
					<td><i>Avihepadnavirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/alignments/internal/avihepadna-eves.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/avihepadna-core.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/avihepadna-surface.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/avihepadna-pol.aln.fna">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>Herpetohepadnavirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/alignments/internal/herpetohepadna-eves.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/herpetohepadna-core.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/herpetohepadna-surface.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/herpetohepadna-pol.aln.fna">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>Metahepadnavirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/alignments/internal/metahepadna-eves.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/metahepadna-core.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/metahepadna-surface.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/metahepadna-pol.aln.fna">FASTA MSA</a></td>
				</tr>

			</tbody>
			
		</table>
		</p>
	   <br>

		<p>
           <strong>Protein level data:</strong>
  		</p>

		<p>
		
		<table>
		
			<thead>
				<tr>
					<th>Taxonomic group</th>
					<th>Core AA</th>
					<th>Surface AA</th>
					<th>Pol AA</th>
				</tr>
			</thead>
		
			<tbody>

				<tr>
					<td><i>Avihepadnavirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/avihepadna-core.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/avihepadna-surface.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/avihepadna-pol.aln.faa">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>Herpetohepadnavirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/herpetohepadna-core.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/herpetohepadna-surface.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/herpetohepadna-pol.aln.faa">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>Metahepadnavirus</i></td>					
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/metahepadna-core.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/metahepadna-surface.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/export/metahepadna-pol.aln.faa">FASTA MSA</a></td>
				</tr>

			</tbody>
			
		</table>


 		</p>

	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>

		<p>

          Multiple sequence alignment constructed in this study are linked together using
          GLUE's <a href="./alignmenttree.html"><b>alignment tree</b></a> data structure.
          Alignments in the project include:

		  <ol>
			<li> A single ‘<a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/alignments/root/hepadnaviridae.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major hepadnavirus lineages
			(including extinct lineages represented only by eHBVs).</li>
			<li> ‘<a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/tree/master/alignments/internal/"><strong>Genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific hepadnavirus genera and eHBV reference sequences.
			<li> ‘<a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/tree/master/alignments/tip"><strong>Tip</strong></a>’
			alignments in which all taxa are derived from a single eHBV lineage.</li>
		  </ol>


 		</p>

	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>

		<p>
          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/tree/master/trees/main-root"><strong>root</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/tree/master/trees/eve-genus"><strong>Genus-level</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/tree/master/trees/eve-orthologs"><strong>eHBV lineage-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


        
	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions in Hepadnaviridae-GLUE</strong>
	   </h3>
	   <hr>


		 <p>
		   The paleovirus component of Hepadnavirus-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of EVE-specific data.
		   These schema extensions are defined in 
           <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/blob/master/glue/build/eve/ehbvSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional tables: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains information pertaining to individual
		   EVE sequences: e.g. species in which they occur, genome assembly version, 
		   genomic location(i.e. scaffold, location coordinates, and orientation).
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains information pertaining to our
		   eHBV reference sequences, which we have constructed in an effort to
		   reconstruct, as closely as possible, the sequences of the progenitor viruses
		   that gave rise to EVEs. 
		</p>








		
		<br>
		<h3>
		  <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
		</h3>
		<hr>

		<p>
	  
		  Lytras S, Arriagada G, and RJ Gifford
		  <b>(2020)</b>
		  <br>
		  Ancient evolution of hepadnaviral paleoviruses and their impact on host genomes.
		  <br>
		  <i>Virus Evolution</i>
		  <a href="https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veab012/6157734" target="_blank">[view]</a>
		  <br>
		  <br>
	 
		  Larsen ON, Wahlberg M, and J. Christensen-Dalsgaard
		  <b>(2020)</b>
		  <br>
		  Amphibious hearing in a diving bird, the great cormorant (Phalacrocorax carbo sinensis).
		  <br>
		  <i>J Exp Biol.</i>
		  <a href="https://jeb.biologists.org/content/223/6/jeb217265.long" target="_blank">[view]</a>
		  <br>
		  <br>


          Prum RO, Berv JS, Dornburg A, Field DJ, Townsend JP, Lemmon EM, Lemmon AR. 
 		  <b>(2015)</b>
		  <br>
		  A comprehensive phylogeny of birds (Aves) using targeted next-generation DNA sequencing.
		  <br>
		  <i>Nature</i>
		  <a href="http://dx.doi.org/10.1038/nature15697" target="_blank">[view]</a>
		  <br>
		  <br>

	 
		  Suh A, Brosius J, Schmitz J, and JO Kriegs.
		  <b>(2013)</b>
		  <br>
		  The genome of a Mesozoic paleovirus reveals the evolution of hepatitis B viruses.
		  <br>
		  <i>Nat Commun.</i> 
		  <a href="https://dx.doi.org/10.1038/ncomms2798" target="_blank">[view]</a>
		  <br>
		  <br>


		  Katzourakis A. and RJ. Gifford
		  <b>(2010)</b>
		  <br>
		  Endogenous viral elements in animal genomes.
		  <br>
		  <i>PLoS Genetics</i>
		  <a href=" https://doi.org/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

	   </p>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE-EVE/">Hepadnavirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



