<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Hepadnavirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Hepadnavirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of hepadnaviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./news.html" class="btn">News</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">


	   <h3>
		 <a id="VirusProject" class="anchor" href="#VirusProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Virus data included in Hepadnavirus-GLUE</strong>
	   </h3>
	   <hr>


		 <p>
		   This page provides background information on the <b>virus-associated</b>
		   data items included in the project  - information about endogenous hepadnaviral elements (eHBVs)
		   can be found <a href="./paleoviruses.html""><b>here</b></a>.
		 </p>


		 <p>
		   <b>Please note</b>: links to files on GitHub are mainly designed to indicate where
		   these files are located within the repository. 
		   To investigate files (e.g. tree files) in the appropriate software context we recommend
		   <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master"><b>downloading the entire repository</b></a>
		   and browsing locally. 	  
		 </p>


		 <p>
		   Those specifically interested in <b>hepatitis B virus (HBV)</b>
		   may want to investigate    
		   <a target="_blank" href="https://github.com/giffordlabcvr/HBV-GLUE"><b>HBV-GLUE</b></a>,
		   and
		   <a target="_blank" href="https://github.com/giffordlabcvr/NCBI-HBV-GLUE"><b>NCBI-HBV-GLUE</b></a>.		   
		 
		   This family of GLUE projects was developed specifically for HBV and incorporates
		   a <b>graphical user interface (GUI)</b>
		   (<a target="_blank" href="https://github.com/giffordlabcvr/HBV-GLUE-WEB"><b>HBV-GLUE-WEB</b></a>),
		   that allows users to browse and interrogate the underlying GLUE database
		   via 'point-and-click' methods.
		 </p>

		
		 <p>
			 The <b>MRC-University of Glasgow Centre for Virus Research</b> hosts an instance of the
			 <a target="_blank" href="http://hbv-glue.cvr.gla.ac.uk/"><b>GUI version of HBV-GLUE</b></a>.


		 </p>

	   <br>
	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hepadnavirus genome features</strong>
	   </h3>
	   <hr>

		<p>
          Hepadnaviruses have enveloped, spherical virions and a small, circular DNA
          genome ~3 kilobases (Kb) in length. The genome is characterised by a highly
          streamlined organization incorporating extensive gene overlap - the open reading
          frame (ORF) encoding the viral polymerase (P) protein occupies most of the genome
          and typically overlaps at least one of the ORFs encoding the core (C), and
          surface (S) proteins. 
 		</p>
		
		<p>
         We defined a
          <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/glue/build/core/hepadnaFeatures.glue"><b>standard set of genome features</b></a>
          for hepadnaviruses and the locations of these genome features
		   on master reference sequences
		   (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/glue/build/core/hepadnaExplicitFeatureLocations.glue">here</b></a>).
		  
		 </p>
		


	   <br>
	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hepadnavirus sequences and sequence-associated data</strong>
	   </h3>
       <hr>


	   <p>
		 The sequence data in this project are organised into multiple distinct sources.
		 Each source contains data in either GenBank XML or plain FASTA format.
		 The type of data is indicated by the name of the source (all GenBank XML sources 
		 contain 'ncbi' in the name).
	   </p>
	   

	   <p>
		 GenBank XML files are imported into this project directly from
		 <b><a target="_blank" href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI GenBank</a></b>
		 using
         <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnavirus-GLUE/blob/master/modules/hepadnaviridaeGenbankXmlPopulator.xml">an appropriately configured</a>
		 version of GLUE's <b>GenBank importer module</b>.
		 The core Hepadnavirus-GLUE project contains a single NCBI-derived source
		 - <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnavirus-GLUE/tree/master/sources/ncbi-refseqs"><b>ncbi-refseqs</b></a> -
		 that contains 'master reference' genome sequences for each hepadnavirus species included in this project.		 		 
	   </p>




	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hepadnavirus reference sequences</strong>
	   </h3>
	   <hr>
		
		 <p>

		   <b>Hepadnavirus-GLUE</b> contains 
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/commandModes/commandMode/root_project_reference"><b>reference sequences</b></a>
		   for all known hepadnavirus species.
		 </p>
		
	   
         <p><img src="./../assets/images/hepadna-hosts-3.jpg" alt="Known hepadnavirus host species" /></p>

		 <blockquote>
				 <p>
				   Left to right: Recent research has identified divergent hepadnaviruses in:
				   (i) <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S1931312817303025">icefish</a> and
				   (ii) <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S1931312817303025">spiny lizards</a>.
				   Viruses closely related to hepatitis B virus, which infects humans, have been identified in
				   a wide range of mammals including
				   (iii) <a target="_blank" href="https://www.pnas.org/content/95/10/5757">woolly monkeys</a> and
				   (iv) <a target="_blank" href="https://www.mdpi.com/1999-4915/11/3/279">duikers</a>.
				 </p>    
		 </blockquote>

	     <br>
	   
		 <p>

		   For each hepadnaviral genus, we have created a 'master' reference sequence, as follows:
		   <ul>
			 <li><b><i>Orthohepadnavirus</i></b>: Hepatitis B virus, strain ayw (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003977"><b>NC_003977</b></a>)</li>
			 <li><b><i>Avihepadnavirus</i></b>: Duck hepatitis B virus, isolate DHBVQCA34 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001344"><b>NC_001344</b></a>)</li>
			 <li><b><i>Herpetohepadnavirus</i></b>: Tibetan frog hepatitis B virus, isolate 243398 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_030446"><b>NC_030446</b></a>)</li>
			 <li><b><i>Metahepadnavirus</i></b>: Bluegill hepatitis B virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_030445"><b>NC_030445</b></a>)</li>
			 <li><b><i>Parahepadnavirus</i></b>: White sucker hepadnavirus, isolate RR173 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_027922"><b>NC_027922</b></a>)</li>
		   </ul>

		 </p>





	   <br>
		
		 <p>

		   Reference sequences are linked to auxiliary data in		 
		   <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/tabular/core/hepadna-ncbi-refseqs-side-data.tsv"><b>tabular format</b></a>.

		 </p>



	   <br>

	   
	   <table>
	
		   <thead>
			   <tr>
				   <th>Parameter</th>
				   <th>Type</th>
				   <th>Definition</th>
			   </tr>
		   </thead>
	
		   <tbody>

			   <tr>
				   <td><i>full_name</i></td>
				   <td>VARCHAR</td>
				   <td>Full name of the virus this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>name</i></td>
				   <td>VARCHAR</td>
				   <td>Abbreviated name of the virus this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>genus</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus genus</td>
			   </tr>

			   <tr>
				   <td><i>clade</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus clade</td>
			   </tr>

			   <tr>
				   <td><i>isolate_name</i></td>
				   <td>VARCHAR</td>
				   <td>Name of the virus isolate this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>host_sci_name</i></td>
				   <td>VARCHAR</td>
				   <td>Species (Latin binomial) virus was isolated from</td>
			   </tr>

			   <tr>
				   <td><i>host_name</i></td>
				   <td>VARCHAR</td>
				   <td>Species (common name) virus was isolated from</td>
			   </tr>

			   <tr>
				   <td><i>length</i></td>
				   <td>INTEGER</td>
				   <td>Length of the sequence</td>
			   </tr>

			   <tr>
				   <td><i>pubmed_id</i></td>
				   <td>INTEGER</td>
				   <td>PubMed ID of manuscript associated with sequence</td>
			   </tr>

			   <tr>
				   <td><i>gb_create_date</i></td>
				   <td>GenBank</td>
				   <td>GenBank creation date of the sequence</td>
			   </tr>

			   <tr>
				   <td><i>gb_update_date</i></td>
				   <td>VARCHAR</td>
				   <td>Date of most recent GenBank update</td>
			   </tr>

			   <tr>
				   <td><i>country</i></td>
				   <td>VARCHAR</td>
				   <td>Country where virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>collection_year</i></td>
				   <td>INTEGER</td>
				   <td>Year virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>collection_month</i></td>
				   <td>VARCHAR</td>
				   <td>Month virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>collection_month_day</i></td>
				   <td>VARCHAR</td>
				   <td>Day of month virus was isolated</td>
			   </tr>


		   </tbody>
		
	   </table>




	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>


		<p>
          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          <a href="./alignmenttree.html">alignment tree</a>
          data structure. Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/alignments/root/hepadnaviridae.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major hepadnavirus lineages</li>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/alignments/internal/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific hepadnavirus genera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/trees/main-root"><strong>root</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/trees/main-genus"><strong>Genus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Project-specific schema extensions</strong>
	   </h3>
	   <hr>


		 <p>
		   Hepadnavirus-GLUE extends GLUE's 
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   through the incorporation of a number of additional fields in the 
		   sequence table, and a project-specific custom table: 'isolate'.
		   These schema extensions are defined <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/glue/build/core/hepadnaSchemaExtensions.glue"><b>here</b></a>.
		   The isolate table is linked to the main 'sequence' table via the sequence ID field.
		   It contains information pertaining to viral isolates,  e.g. species sampled, date and location of sample.
		 </p>


	   <br>

	   <br>
	   
       <!-- <p><img src="./../assets/images/virus-project-images.jpg" alt="Liver punch / Liver disease / Hepatitis B virus" /></p> -->

 

	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/">Hepadnavirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



