<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <meta charset="UTF-8">
    <title>Hepadnaviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="website/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/github-light.css" media="screen">
    
  </head>

  <body>
    <section class="page-header">
    
      <h1 class="project-name">Hepadnaviridae-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of hepadnaviruses.</h2>
      <a href="./website/viruses.html" class="btn">Viruses</a>
      <a href="./website/paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">

	   <h3>
		 <a id="description" class="anchor" href="#description" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Description</strong>
	   </h3>




		 <p>
		   This is <b>Hepadnaviridae-GLUE</b>, a GLUE
		   project supporting comparative genomic and evolutionary analysis of
		   <a target="_blank" href="https://viralzone.expasy.org/9">hepadnaviruses</a>
		   (family <i>Hepadnaviridae</i>).
		 </p>


		 <p>
		   <b>Hepadnaviridae-GLUE</b> contains a richly annotated sequence
		   dataset for these viruses, comprised of both 
		   <a href="./website/viruses.html"><b>viral sequences</b></a>
		   and
		   <a href="./website/paleoviruses.html"><b>endogenous viral elements (EVEs)</b></a>.
		   
		 </p>

		 <br>

        <p><img src="./website/images/hepadnavirus-header.png" alt="Species known to be afflicted by hepadnaviruses" /></p>



<blockquote>
		<p>
           <b>Timeline of hepadnavirus evolution</b>. 
              The inset panel shows the evolutionary relationships of hepadnavirus genera,
              the larger tree is a time-calibrated evolutionary tree of vertebrates.
              Geological eras are indicated by background shading. The scale bar shows
              time in millions of years before present.           
              For details see <a target="_blank" href="https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veab012/6157734">Lytras <i>et al</i></a> (2020).	
              Mya=Million years ago.
        </p>
         
</blockquote>
		 


		 <br>

		 <p>
		 
		 
          <b>There are a wide variety of ways in which this resource can be used</b>:
           
 		  <ul>
 		 
		   <li> If you're interested in any particular virus within the family <i>Hepadnaviridae</i>,
		   you can use Hepadnaviridae-GLUE as an efficient means of investigating this species in depth.
		   </li>
		   <br>
		   
		   <li> For those interested in <b>paleovirology</b> and <b>endogenous viral elements (EVEs)</b>,
		   Hepadnaviridae-GLUE provides a source of systematically organised information about
		   <a href="./website/paleoviruses.html"><b>EVEs derived from hepadnaviruses</b></a>.
		   <br>
		   <br>

		   <u><b>For those specifically interested in <b>hepatitis B virus (HBV)</b>, we suggest to investigate
		   our separate
		   <a target="_blank" href="http://hbv-glue.cvr.gla.ac.uk/"><strong>HBV-GLUE</strong></a> resource.</b></u>

		   </li>


		   <br>
		   <li> If you're interested in exploring <b>hepadnavirus diversity</b> 
		   in <b>metagenomic</b> datasets, the data included in this project 
		   may assist you in your analysis, as explained
		   <a href="./website/paleoviruses.html"><b>here</b></a>.
		   </li>
		   <br>
		   
		 </ul>
		  
		 </p>


	   <h3>
	   <a id="WhatGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What is a GLUE project?</strong>
	   </h3>
       <hr>


	   <p>
		 <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a> is an open, integrated
		 software toolkit that provides functionality for storage and interpretation of
		 sequence data. 
	   </p>

	   <p>
		 <b>GLUE</b> supports the development of <b>“projects”</b> containing the data items
		 required for comparative genomic analysis 
		 (e.g.  sequences, multiple sequence alignments, genome feature annotations,
		 and other sequence-associated data). 
	   </p>

	   <p>
		 Projects are loaded into the <b>GLUE "engine"</b>, creating a relational database
		 that represents the semantic relationships between data items.
		 This provides a robust foundation for the implementation of <b>systematic
		 comparative analyses</b> and the development of <b>sequence-based resources</b>. 
	   </p>

	   <p>		 
		  The core schema of this database can be extended to accommodate
		  the idiosyncrasies of different projects, and
		  GLUE provides a scripting layer (based on JavaScript) for developing custom
		  analysis tools. 
	   </p>

	   <p>
		  Hosting of GLUE projects in an online version control system (e.g. GitHub) provides
		  a mechanism for their stable, collaborative development.
	   </p>
	   
	   

		   
		 <p>
		   Some examples of '<b>sequence-based resources</b>' built for viruses using GLUE include:

		 </p>
		   
		   

		 <p>

		 <ul>
		 		 
		   <li> 
			 <a target="_blank" href="http://cov-glue.cvr.gla.ac.uk/"><b>COV-GLUE</b></a>: 
			 A GLUE resource for tracking genetic variation in SARS-COV2.
			 CoV-GLUE contains a database of amino acid replacements, insertions and
			 deletions which have been observed in GISAID hCoV-19 sequences sampled from the pandemic
		   </li>
		   <br>
		   
		   <li>
			 <a target="_blank" href="http://rabv-glue.cvr.gla.ac.uk/"><b>RABV-GLUE</b></a>:
			 Tailored toward epidemiological tracking of rabies virus (RABV).
			 Includes a database of RABV sequences and metadata from NCBI, updated daily and arranged into major and minor clades, and
			 an analysis tool providing genotyping, analysis and visualisation of submitted FASTA sequences.
 		   </li>
 		   <br>
		   
		   <li>
			 <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>HCV-GLUE</b></a>:
			 This GLUE resource aims to support analysis of drug resistance and vaccine
			 escape in hepatitis C virus (HCV).
			 A database of HCV sequences and metadata from NCBI, updated daily and arranged
			 into clades (genotypes, subtypes). As well as pre-built multiple-sequence
			 alignments of NCBI sequences, it includes an analysis tool providing genotyping,
			 drug resistance analysis and visualisation of submitted FASTA sequences.
           </li>
	   
		 </ul>

		   
		 </p>
	   



	   


	   <h3>
	   <a id="WhyGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What does building the Hepadnaviridae-GLUE project offer?</strong>
	   </h3>

		 
 
	   <p>
		 
		  Hepadnaviridae-GLUE offers a number of advantages for performing comparative sequence
		  analysis of hepadnaviruses:

		 <ol>
		   <li> <strong>Standardisation of the genomic co-ordinate space</strong>. GLUE
		   projects allow all sequences to utilise the coordinate space of a chosen 
		   reference sequence. Contingencies associated with insertions and deletions
		   (indels) are handled in a systematic way.
		   </li>
		   <li> <strong>Predefined, fully annotated reference sequences</strong>:
		   This project includes fully-annotated reference sequences for major lineages
		   within the <i>Hepadnaviridae</i> family.
		   </li>
		   <li> <strong>Alignment trees</strong>:
		   GLUE allows linking of alignments constructed at distinct taxonomic levels
		   via an "alignment tree" data structure. In the alignment tree, each alignment
		   is constrained to a standard reference sequence, thus all multiple sequence
		   alignments are linked to one another via a standardised coordinate system.
		   </li>
		 </ol>

	   </p>

	   <h3>
		 <a id="Related resources" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related resources</strong>
	   </h3>
 
	   <p>
		 We have also used the GLUE software environment to construct a sequence-based
		 resource specifically focussed on <strong>hepatitis B virus</strong> (HBV).
		 <strong></strong> 
		 <a target="_blank" href="https://github.com/giffordlabcvr/HBV-GLUE"><strong>HBV-GLUE</strong></a>
		 is an open resource, hosted on GitHub, which is aimed at investigators interested
		 in the genetic diversity of HBV and its evolution in response to
		 vaccine and drug treatment.
	   </p>
 
	   <p>
		 A web user interface to  <strong>HBV-GLUE</strong> can be found 		 
		 <a target="_blank" href="http://hbv-glue.cvr.gla.ac.uk/"><strong>here</strong></a>.
	   </p>
 



	   <h3>
	   <a id="glueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>GLUE project</strong>
	   </h3>

      	   
	   <p>
	
         On computers with <a href="http://glue-tools.cvr.gla.ac.uk/#/installation">GLUE installed</a>,
         the <b>Hepadnaviridae-GLUE</b> project can be instantiated by navigating to the project folder,
         initiating GLUE, and issuing the following command in the GLUE shell:
         
  <pre>
  <code>Mode path: /
  GLUE> run file hepadnaviridaeProject.glue</code></pre>
		 
	   </p>




	   <h3>
		 <a id="contributors" class="anchor" href="#contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Contributors</strong>
	   </h3>


		 <p><a href="https://giffordlabcvr.github.io/virus-evolution/index.html" target="_blank">Robert J. Gifford</a> (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)</p>
		 <p>Spyros Lytras (<a href="mailto:s.lytras.1@research.gla.ac.uk">s.lytras.1@research.gla.ac.uk</a>)</p>		 


	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>


		<p>
		
		
		Lytras S, Arriagada G, and RJ Gifford
		<b>(2020)</b>
		<br>
		Ancient evolution of hepadnaviral paleoviruses and their impact on host genomes.
		<br>
		<i>Virus Evolution</i>
		<a href="https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veab012/6157734" target="_blank">[view]</a>
		<br>
		<br>

		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>

		Zhu H, Dennis T, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		<a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		<br>
		<br>

		Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		<b>(2018)</b>
		<br>
		Nomenclature for endogenous retrovirus (ERV) loci.
		<br>
		<i>Retrovirology</i>
		<a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		<br>
		<br>

		Katzourakis A. and RJ. Gifford
		<b>(2010)</b>
		<br>
		Endogenous viral elements in animal genomes.
		<br>
		<i>PLoS Genetics</i>
		<a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

 		</p>



	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/DIGS-tool">Hepadnaviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>
