<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Hepadnaviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Hepadnaviridae-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of hepadnaviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./project.html" class="btn">Viruses</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">

	   <h3>
		 <a id="'Alignment Trees' in GLUE" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>The GLUE alignment tree structure</strong>
	   </h3>



		<p>
          GLUE projects have the option of using a structure called an 'alignment tree',
          which links together nucleotide homologies in a useful, evolution-oriented way.
          An alignment tree is built by first creating constrained alignments for
          each of the established clades for the viruses of interest. This can be done
          within GLUE, or externally using any favoured approach to MSA construction. 
 		</p>



		<p>
         Where a parent-child relationship between two clades exists within the
          evolutionary hypothesis, a special relational link is introduced between the
          corresponding pairs of Alignment objects. Sequence objects are then assigned
          to clades by adding them as members of the corresponding alignment.
		</p>



		<p>
          We created a set of linked alignments that represented distinct
          taxonomic groups and levels within the family Hepadnaviridae. For the lower
          taxonomic levels (i.e. within and below genus level) we aligned complete coding
          sequences.
          
		  
		</p>

 
        <p><img src="./images/alignmentTreeConcept.jpg" alt="Alignment tree concept" /></p>

 		

		<p>       
          For higher taxonomic levels we aligned the polymerase or RT genes. 
          We used GLUE to link these alignments, via a set of common reference sequences.
          The root alignment contains all reference sequences, whereas all children of the
          root inherit at least one reference from their immediate parent. Thus, all alignments are linked to one another via our chosen set of references.
 		</p>
s

	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/">Hepadnaviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



