<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Hepadnaviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>


  <body>
    <section class="page-header">
      <h1 class="project-name">Hepadnaviridae-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of hepadnaviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>



    <section class="main-content">


	   <h3>
		 <a id="'Highlights" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hepadnaviridae-GLUE: Highlights</strong>
	   </h3>
	   <hr>

		<p>
          These highlights pages aim to provide a brief overview of selected data items
          contained within the <b>Hepadnaviridae-GLUE</b> project, and to illustrate
          how it can be used to implement reproducible comparative analyses. 

.
 		</p>



	   <h3>
		 <a id="Avi.1.Neoaves" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.1-Neoaves</strong>
	   </h3>
	   <hr>



		<p>
          First eHBV to be reported. Very ancient origin - found in many species.
 		</p>




        <p><img src="./images/ehbv-avi.1-neoaves.png" alt="Background images for Avi.28-Alauda" /></p>
 		


<blockquote>
		  Left panel: distribution of Avi.1.Neoaves insertions. 
		  The insertion is absent in ducks and chickens - the 'Galloanserae' -
		  which are 
		  Analysis using Hepadnaviridae-GLUE has revealed that at least four distinct
          subgroups of avihepadnavirus have existed during hepadnavirus evolution 
</blockquote>



		<p>
           Raw data in tabular format are can be found at the following links/directories: 
		  <ul>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-amdo-refseqs-side-data.tsv">Sphenodon ortholog</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-erythyro-refseqs-side-data.tsv">Avian consensus</a></li>
		  </ul>

  		</p>

  		


 	   <br>
	   <h3>
		 <a id="Avi.14-Gavia-Avi.15-Gavia" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.14-Gavia and Avi.15-Gavia</strong>
	   </h3>
	   <hr>

		<p>
          These two eHBV loci in the genome of the common loon (<i>Gavia immer</i>) 
          derive from germline incorporation events involving
          ancient avihepadnaviruses.
          The viruses involved belong to a characteristic lineage, which we've labeled
          'clade II'. So far these viruses are only known from eHBV sequences, and it
          remains unknown when any modern representatives still circulate in birds.
		</p>

        <p><img src="./images/ehbv-avi.14+15-gavia.png" alt="Background images for Avi.14-Gavia and Avi.15-Gavia" /></p>
 		
 		


<blockquote>
		<p>
		  Left to right: (i) The common loon (<i>Gavia immer</i>) contains two eHBV insertions derived from clade II aviheapdnaviruses.
		  (ii) Emperor penguins also contain clade II avihepadnaviruses, indicating their biogeographic host range could extend to the Antarctic;    
		  eHBV insertions derived from clade II were also identified in (iii) Great crested grebe (<i>Podiceps cristatus</i>) and; (iv) barn owl (<i>Tyto alba</i>) 
		</p>        
</blockquote>
	
 

		<p>
          The two insertions are similar in that they both span near-complete hepadnaviral
          genomes. Remarkably, however, they appear to have been acquired in entirely
          different germline integration events.
          
          In our recent paper we show that. consistent with this, both insertions show
          a greater degree of divergence in a highly region of the Surface protein associated
          with binding to host receptors than they do in other genomic regions.
 		</p>



	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/sources/fasta-refseqs-ehbv/ehbv-avi.14-gavia.fna">Avi.14-Gavia nucleotide sequence</a></li>
		   <li> <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/sources/fasta-refseqs-ehbv/ehbv-avi.15-gavia.fna">Avi.15-Gavia nucleotide sequence</a></li>
		 </ul>

	   </p>
 
 	   <br>
	   <h3>
		 <a id="ehbv-avi.27-Phalacrocoracidae" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.27-Phalacrocoracidae</strong>
	   </h3>
	   <hr>



		<p>
          A multicopy eHBV LINEAGE.

 		</p>


        <p><img src="./images/ehbv-avi.27-phalacrocoracidae.png" alt="Background images for Avi.28-Alauda" /></p>
 		



<blockquote>

		<p>
		  Cormorants have a multi-copy eHBV lineage in th- left to right:
		</p>
         
</blockquote>


 		

 	   <br>
	   <h3>
		 <a id="'Avi.28-Alauda" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.28-Alauda</strong>
	   </h3>
	   <hr>

		<p>


          All extant avihepadnaviruses fall into one subgroup (which we labeled 'IV' in our study). 

  		</p>



		<p>

          Avi.28.Alauda is the most closely related EVE yet to extant avihepdanviruses. 
          It groups basal to the clade IV avihepadnaviruses, and encodes a Surface 
          protein that is relatively closely related to those found among extant 
          avihepadnaviruses.

 		</p>


        <p><img src="./images/ehbv-avi.28-alauda.png" alt="Background images for Avi.28-Alauda" /></p>
 		


<blockquote>
		Eurasian skylark (<i>Alauda arvensis</i>)       
</blockquote>
	
  		

		<p>

          Avi.28.Alauda is the most closely related EVE yet to extant avihepdanviruses. 
          It groups basal to the clade IV avihepadnaviruses, and encodes a Surface 
          protein that is relatively closely related to those found among extant 
          avihepadnaviruses.

 		</p>



	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/sources/fasta-refseqs-ehbv/ehbv-avi.14-gavia.fna">Avi.28-Gavia nucleotide sequence</a></li>
		   <li> <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/sources/fasta-refseqs-ehbv/ehbv-avi.15-gavia.fna">Hepadnaviridae pylogeny with EVEs</a></li>
		 </ul>

	   </p>



	   <h3>
		 <a id="'Meta.1.Sauria" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Meta.1-Sauria</strong>
	   </h3>
	   <hr>


		<p>
		  Meta.1.Sauria is - we think - the oldest EVE ever described. We identified
		  putatively orthologous copies in a lizard - the tuatara (Sphenodon) -
		  as well as in multiple avian species.   
   		</p>


		<p>
 		  This insertion also provides important new insights into the evolution of
		  hepadnaviruses.
          It belongs to the recently described <i>Metahepadnavirus</i> genus.
		  So far, metahepadnaviruses have only been identifiied in fish.
		  Unexpectedly, these fish viruses were found to group relatively closely with
		  mammalian hepadnaviruses (genus <i>Orthohepadnavirus</i>). This seemed to imply 
		  a transfer of viruses between fish and mammals! 
   		</p>



        <p><img src="./images/ehbv-meta.1-sauria.png" alt="Species containing Meta.1-Sauria insertions in their genomes" /></p>
 		

<blockquote>
		  Species containing Meta.1-Sauria insertions in their genomes - left to right:
</blockquote>
	



		<p>
           Raw data in tabular format are can be found at the following links/directories: 
		  <ul>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-amdo-refseqs-side-data.tsv">Sphenodon ortholog</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-erythyro-refseqs-side-data.tsv">Avian consensus</a></li>
		  </ul>

  		</p>

  		







	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/">Hepadnaviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



