<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Hepadnaviridae-GLUE by giffordlabcvr: Highlights</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>


  <body>
    <section class="page-header">
      <h1 class="project-name">Hepadnaviridae-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of hepadnaviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>



    <section class="main-content">


	   <h3>
		 <a id="'Highlights" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hepadnaviridae-GLUE: Highlights</strong>
	   </h3>
	   <hr>

		<p>
          These highlights pages aim to provide a brief overview of selected data items
          contained within the <b>Hepadnaviridae-GLUE</b> project, and to illustrate
          how it can be used to implement reproducible comparative analyses. 
 		</p>



	   <h3>
		 <a id="Avi.1.Neoaves" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.1-Neoaves</strong>
	   </h3>
	   <hr>



		<p>
          eHBV-Avi.1-Neoaves was the first hepadnavirus-derived EVE to be described

          (<a target="_blank" href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191">Katzourakis and Gifford, 2010</a>;
          <a target="_blank" href="https://www.nature.com/articles/ncomms2798">Suh et al, 2013</a>).
          It is located in an intron of the
          <a target="_blank" href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=FRY"><b>FRY Microtubule Binding Protein</b></a></li>
       
          gene and derives from a germline incorporation
          event involving an ancient <b>avihepadnavirus</b>.
          This event predates the divergence of the avian clade <b>Neoaves</b>, which includes 
          all modern birds except the Galloanserae (ducks and chickens) and Palaeognathae.
          Consequently, an Avi.1-Neoaves insertions are present in multiple avian species.
 		</p>




        <p><img src="./images/ehbv-avi.1-neoaves.jpg" alt="Background images for Avi.28-Alauda" /></p>
 		


<blockquote>
		  Left panel: the clade 'Neoaves' includes almost all modern birds, including (clockwise)
		  flamingos (order Phoenicopteriformes), parrots (order Psittaciformes),
		  songbirds (order Passeriformes) and all birds of prey.
		  Right panel: Analysis using Hepadnaviridae-GLUE has revealed that at least four distinct
          clades (I-IV) of avihepadnavirus have existed during hepadnavirus evolution.

</blockquote>


		<p>
		  Links to related data items: 

		  <ul>
			<li> <a target="_blank" href="../sources/fasta-refseqs-ehbv/ehbv-avi.1-neoaves.fna">eHBV-Avi.1-Neoaves nucleotide consensus sequences</a></li>
		    <li> <a target="_blank" href="./images/avihepadna-phylogeny.jpg">Avihepadnaviridae phylogeny with EVEs</a></li>
		    <li> <a target="_blank" href="./images/avian-phylogeny-part1.jpg">Avian phylogeny (part 1)</a> (taken from <a target="_blank" href="https://dx.doi.org/10.1038/nature15697">Prun <i>et al</i></a>)</li>
		    <li> <a target="_blank" href="./images/avian-phylogeny-part2.jpg">Avian phylogeny (part 2)</a> (taken from <a target="_blank" href="https://dx.doi.org/10.1038/nature15697">Prun <i>et al</i></a>)</li>
		  </ul>

  		</p>



 	   <br>
	   <h3>
		 <a id="Avi.14-Gavia-Avi.15-Gavia" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.14-Gavia and Avi.15-Gavia</strong>
	   </h3>
	   <hr>

		<p>
          These two eHBV loci in the genome of the common loon (<i>Gavia immer</i>) 
          derive from germline incorporation events involving
          ancient avihepadnaviruses.
          The viruses involved belong to a characteristic lineage, which we've labeled
          'clade II'. So far these viruses are only known from eHBV sequences, and it
          remains unknown whether any modern representatives still circulate in birds.
		</p>

        <p><img src="./images/ehbv-avi.14+15-gavia.png" alt="Background images for Avi.14-Gavia and Avi.15-Gavia" /></p>
 		
 		


<blockquote>
		<p>
		  Left to right: (i) The common loon (<i>Gavia immer</i>) contains two eHBV insertions derived from clade II aviheapdnaviruses.
		  (ii) Emperor penguins also contain clade II avihepadnaviruses, indicating their biogeographic host range could extend to the Antarctic;    
		  eHBV insertions derived from clade II were also identified in (iii) Great crested grebe (<i>Podiceps cristatus</i>) and; (iv) barn owl (<i>Tyto alba</i>) 
		</p>        
</blockquote>
	
 

		<p>
          The two insertions are similar in that they both span near-complete hepadnaviral
          genomes. Remarkably, however, they appear to have been acquired in entirely
          different germline integration events.
          
          In our recent paper we show that - consistent with this - both insertions show
          a greater degree of divergence in a region of the Surface protein associated
          with binding to host receptors than they do in other genomic regions.
 		</p>



	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="../sources/fasta-refseqs-ehbv/ehbv-avi.14-gavia.fna">Avi.14-Gavia nucleotide sequence</a></li>
		   <li> <a target="_blank" href="../sources/fasta-refseqs-ehbv/ehbv-avi.15-gavia.fna">Avi.15-Gavia nucleotide sequence</a></li>
		   <li> <a target="_blank" href="./images/cladeII_var_fn.png">Similarity plots comparing clade II eHBV sequences</a></li>
		 </ul>

	   </p>
 
 	   <br>
	   <h3>
		 <a id="ehbv-avi.27-Phalacrocoracidae" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.27-Phalacrocoracidae</strong>
	   </h3>
	   <hr>



		<p>
          Avi.27-Phalacrocoracidae is a multicopy eHBV lineage.
          We identified approximately 300 copies of this avihepadnavirus derived element
          in the cormorant germline.
          All of these elements are quite short, spanning around 350 nucleotides of the polymerase gene.
          Analysis of flanking DNA regions suggested that the insertion has been mobilised
          within the cormorant germline by transposable elements (TEs).

 		</p>


        <p><img src="./images/ehbv-avi.27-phalacrocoracidae.png" alt="Background images for Avi.28-Alauda" /></p>
 		



<blockquote>

		<p>
          Cormorants have many anatomical and physiological adaptations for diving,
          including amphibious hearing.
		  Left to right: (i) and (ii) Great cormorant (<i>Phalacrocorax carbo</i>);
		  (iii) Brandt's cormorant  (<i>Phalacrocorax penicillatus</i>); 
		  (iv) Cormorant fishing is a traditional fishing method in which fishermen use
		  trained cormorants to fish in rivers. Historically, cormorant fishing has taken
		  place in Japan and China, as well as some European countries.
		</p>
         
</blockquote>

		<p>
  
          Phylogenies indicate that the initial germline incorporation event that gave rise
          to this multicopy eHBV lineage predates the diversification of cormorants.
          How did so many distinct eHBV/TE insertions get fixed in the cormorant germline?
          Possibly, the capture of EVE sequences offered a selective advantage to TE lineages
          that allowed them to proliferate so widely that numerous copies were fixed only by chance.
          Alternatively, TE sequences containing hepadnavirus-derived DNA might,
          for some reason, be more likely to be fixed. 
 
 		</p>

 
		<p>
          Numerous lines of evidence indicate that TEs play a role in rewiring gene
          regulatory networks. This process may well underlie many of the evolutionary
          adaptations seen in vertebrates. One possible explanation for the 
          fixation of so many eHBV-avi.28 copies in cormorants might be if they, or the TEs
          they are embedded within, have played a role in facilitating the adaptation 
          of these birds during their evolution.
 		</p>



	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/sources/fasta-refseqs-ehbv/ehbv-avi.27-phalacrocoracidae.fna">Avi.27-Phalacrocoracidae consensus sequence</a></li>
		   <li> <a target="_blank" href="./images/avihepadna-phylogeny.jpg">Avihepadnaviridae pylogeny with EVEs</a></li>
		 </ul>

	   </p>
		

 	   <br>
	   <h3>
		 <a id="'Avi.28-Alauda" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Avi.28-Alauda</strong>
	   </h3>
	   <hr>


		<p>
		  We identified a full-length insertion in the Eurasian skylark (<i>Alauda arvensis</i>) 
		  genome that shows a higher level of relatedness to modern hepadnaviruses
		  than does any previously reported eHBV. 
          All extant avihepadnaviruses fall into one subgroup (which we labeled 'IV' in our study). 
          Avi.28.Alauda groups as a basal member of this clade.
 		</p>


        <p><img src="./images/ehbv-avi.28-alauda.png" alt="Background images for Avi.28-Alauda" /></p>
 		


<blockquote>
		Eurasian skylark (<i>Alauda arvensis</i>)       
</blockquote>
	
  	
	   <p>
		  Links to related data items: 
		 <ul>
		   <li> <a target="_blank" href="../sources/fasta-refseqs-ehbv/ehbv-avi.28-alauda.fna">eHBV-Avi.28-Alauda nucleotide sequence</a></li>
		   <li> <a target="_blank" href="./images/avihepadna-phylogeny.jpg">Avihepadnaviridae phylogeny with EVEs</a></li>
		 </ul>

	   </p>



	   <h3>
		 <a id="'Meta.1.Sauria" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Meta.1-Sauria</strong>
	   </h3>
	   <hr>


		<p>
		  Meta.1.Sauria is - we think - the oldest EVE ever described. We identified
		  putatively orthologous copies in a lizard - the tuatara (<i>Sphenodon</i>) -
		  as well as in multiple avian species. This implies that germline integration 
		  occurred prior to the divergence of these lineages in the late Paleozoic Era 
		  (~300 million years ago).
   		</p>


		<p>
 		  This insertion also provides important new insights into the evolution of
		  hepadnaviruses.
          It belongs to the recently described <i>Metahepadnavirus</i> genus.
		  So far, metahepadnaviruses have only been identifiied in fish.
		  Unexpectedly, these fish viruses were found to group relatively closely with
		  mammalian hepadnaviruses (genus <i>Orthohepadnavirus</i>). This seemed to imply 
		  a transfer of viruses between fish and mammals! 
   		</p>



        <p><img src="./images/ehbv-meta.1-sauria.png" alt="Species containing Meta.1-Sauria insertions in their genomes" /></p>
 		

<blockquote>
		  Metahepadnaviruses are a recently discovered group of hepadnaviruses left to right:
		  (i) White sucker ( ) - one of the fish species in which metahepadnaviruses were first identfied;
		  (i) eHBVs derived from metahapadnaviruses were identified in the tuatara (<i>Sphenodon punctatus</i>) - a highly divergent reptile;
		  (iii) Tuataras are the only surviving representatives of the ancient reptile lineage that contained 
		  the marine ichthyosaurs and pleisiosaurs;
		  (iv) Meta.1-Sauria insertions have also been preserved in some avian genomes.
		  Emperor penguins contain a relatively well-preserved ortholog.
		  
</blockquote>
	



		<p>
           Raw data in tabular format are can be found at the following links/directories: 
		  <ul>
		    <li> <a target="_blank" href="../sources/fasta-refseqs-ehbv/ehbv-meta.1-neoaves.fna">Avian consensus</a></li>
			<li> <a target="_blank" href="../sources/fasta-digs-ehbv/ehbv-meta.1-Sphenodon_punctatus.fas">Sphenodon ortholog</a></li>
		  </ul>

  		</p>
		
		
		<br>
		<h3>
		  <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
		</h3>
		<hr>

		<p>
	  
		  Lytras S, Arriagada G, and RJ Gifford
		  <b>(2020)</b>
		  <br>
		  Ancient evolution of hepadnaviral paleoviruses and their impact on host genomes.
		  <br>
		  <i>Virus Evolution</i>

		  <a href="https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veab012/6157734" target="_blank">[view]</a>
		  <br>
		  <br>
	 
		  Larsen ON, Wahlberg M, and J. Christensen-Dalsgaard
		  <b>(2020)</b>
		  <br>
		  Amphibious hearing in a diving bird, the great cormorant (Phalacrocorax carbo sinensis).
		  <br>
		  <i>J Exp Biol.</i>

		  <a href="https://jeb.biologists.org/content/223/6/jeb217265.long" target="_blank">[view]</a>
		  <br>
		  <br>

	 
		  Suh A, Brosius J, Schmitz J, and JO Kriegs.
		  <b>(2013)</b>
		  <br>
		  The genome of a Mesozoic paleovirus reveals the evolution of hepatitis B viruses.
		  <br>
		  <i>Nat Commun.</i> 
		  <a href="https://www.nature.com/articles/ncomms2798" target="_blank">[view]</a>
		  <br>
		  <br>


		  Katzourakis A. and RJ. Gifford
		  <b>(2010)</b>
		  <br>
		  Endogenous viral elements in animal genomes.
		  <br>
		  <i>PLoS Genetics</i>
		  <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

	   </p>







	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/">Hepadnaviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



