<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-127420264-1"></script>
	<script>
	  window.dataLayer = window.dataLayer || [];
	  function gtag(){dataLayer.push(arguments);}
	  gtag('js', new Date());
	  gtag('config', 'UA-127420264-1');
	</script>


    <meta charset="UTF-8">
    <title>Hepadnaviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Hepadnaviridae-GLUE</h1>
      <h2 class="project-tagline">A sequence data resource for hepadnaviruses.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="./project.html" class="btn">Viruses</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous hepadnavirus (eHBV) data</strong>
	   </h3>

		<p>
          We have used GLUE to organise the fossil record of hepadnaviruses.
          You can access these data
          <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master">in this repository</a>. 
          This website provides quick links to specific data items. 

		</p>


	   <h3>
		 <a id="eHBV reference sequences and data" class="anchor" href="#Ref_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>eHBV reference sequences and data</strong>
	   </h3>

		<p>
           We reconstructed reference sequences for ECVs by aligning eHBV sequences that
           derive from the same ancestral germline integration event.
 		</p>
 		
		
		<p>
           Reference sequences were constructed by first constructing alignments of eHBV
           sequences derived from the same initial germline colonisation event - i.e.
           orthologous elements in distinct species, and paralogous 
           elements that have arisen via intragenomic duplication of eHBV sequences.
  		</p>

		<p>
           <strong>Nucleotide level data:</strong>
  		</p>


		<table>
		
			<thead>
				<tr>
					<th>Taxonomic group</th>
					<th>Tabular data</th>
					<th>Full-length eHBV nucleotides</th>
					<th>Full-length eHBV</th>
				</tr>
			</thead>
		
			<tbody>

				<tr>
					<td><i>Genus Circovirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/tabular/eve/ecv-circo-refseqs-side-data.tsv"><i>2020-07-15</i></a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/alignments/internal/circovirus-genome.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/sources/fasta-refseqs-circo-ecv">Individual FASTA</a></td>
				</tr>

				<tr>
					<td><i>Genus Cyclovirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/tabular/eve/ecv-cyclo-refseqs-side-data.tsv"><i>2020-07-15</i></a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/alignments/internal/cyclovirus-genome.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/sources/fasta-refseqs-cyclo-ecv">Individual FASTA</a></td>
				</tr>

				<tr>
					<td><i>Unclassified CRESS</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/tabular/eve/ecv-cress-refseqs-side-data.tsv"><i>2020-07-15</i></a></td>
					<td><a href="">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/sources/fasta-refseqs-cress-ecv">Individual FASTA</a></td>
				</tr>

			</tbody>
			
		</table>

		<p>
           <strong>Protein level data:</strong>
  		</p>
		
		<table>
		
			<thead>
				<tr>
					<th>Taxonomic group</th>
					<th>Rep protein</th>
					<th>Rep protein</th>
					<th>Cap protein</th>
					<th>Cap protein</th>
				</tr>
			</thead>
		
			<tbody>

				<tr>
					<td><i>Genus Circovirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/export/ecv-circo-rep.polypeptide.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/export/ecv-circo-rep-polypeptide-fasta">Individual FASTA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/export/ecv-circo-cap.polypeptide.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/export/ecv-circo-cap-polypeptide-fasta">Individual FASTA</a></td>
				</tr>


				<tr>
					<td><i>Genus Cyclovirus</i></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/export/ecv-circo-rep.polypeptide.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/export/ecv-circo-rep-polypeptide-fasta">Individual FASTA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/export/ecv-circo-cap.polypeptide.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/export/ecv-circo-cap-polypeptide-fasta">Individual FASTA</a></td>
				</tr>


			</tbody>
			
		</table>








	   <h3>
		 <a id="Multiple sequence alignments and phylogenetic trees" class="anchor" href="#AlignmentsAndTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments and phylogenetic trees</strong>
	   </h3>

		<p>
          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          ‘<a href="./alignment tree concept small.png">alignment tree</a>’
          data structure. Alignments in the project include:
          <br> 
          <br> (i) A single ‘<a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/alignments/root/cress-root.fna"><strong>root</strong></a>’
          alignment constructed to represent proposed
          homologies between representative members of major CRESS virus lineages
          (including extinct lineages represented only by ECVs).
          <br> (ii) ‘<a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/alignments/tip"><strong>genus-level</strong></a>’
          alignments constructed to represent proposed homologies between the genomes of
          representative members of specific CRESS virus genera and eHBV reference sequences.
          <br> (iii) ‘<a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/alignments/tip"><strong>Tip</strong></a>’
          alignments in which all taxa are derived from a single virus species or represent 
          distinct members of a single eHBV lineage.
          
 		</p>


		<p>
          We have used GLUE to set-up an automated process for deriving 
          midpoint rooted, annotated trees for each of the alignments included in 
          our alignment tree. 
          
 		</p>



	   <h3>
		 <a id="Raw eHBV sequences and data" class="anchor" href="#Raw_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw eHBV sequences and data</strong>
	   </h3>

		<p>
          These are the raw data generated by
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>.
          The tabular files contain information about the genomic location of each EVE. 
          EVEs were classified by comparison to a polypeptide sequence reference library
          designed to represent the known diversity of CRESS viruses - this includes extinct 
          lineages represented only by endogenous viral elements (EVEs).
 		</p>

		<p>
          These data were obtained via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI genomes</a>
          (2020-07-15).
		</p>



		<table>
		
			<thead>
				<tr>
					<th>Taxonomic Group</th>
					<th>eHBV locus data in tabular format</th>
					<th>Individual FASTA files</th>
				</tr>
			</thead>
		
			<tbody>

				<tr>
					<td>Genus <i>Circovirus</i></a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/tabular/eve/ecv-circo-side-data.tsv">2020-07-15</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/sources/fasta-digs-circo-ecv">2020-07-15</a></td>
				</tr>


				<tr>
					<td>Genus <i>Cyclovirus</i></a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/tabular/eve/ecv-cyclo-side-data.tsv">2020-07-15</a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/sources/fasta-digs-cyclo-ecv">2020-07-15</a></td>
				</tr>

				<tr>
					<td>Unclassified <i>CRESS</i></a></td>
					<td><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/blob/master/tabular/eve/ecv-cress-side-data.tsv">2020-07-15</a></td>
					<td> <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master/sources/fasta-digs-cress-ecv">2020-07-15</a></td>
				</tr>

			</tbody>
			
		</table>






	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for ECVs</strong>
	   </h3>

		<p>
          We have applied a systematic approach to naming eHBV, following a convention 
          developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>

		<p>
          The first component is the classifier ‘eHBV’ (endogenous CRESS virus element). 
		</p>

		<p>
          The second component is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of eHBV group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
          Where an eHBV lineage has been established via intragenomic amplication an
          incorporated eHBV sequence, distinct duplicates are identified by numbers after 
          the decimal point.        

		</p>
		
		<p>
          The third component of the ID defines the set of host species in which the ortholog occurs.
 		</p>



	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>


		Dennis TPW, de Souza WM, Marsile-Medun S, Singer JB, Wilson SJ, and RJ Gifford 
		<b>(2019)</b>
		<br>
		The evolution, distribution and diversity of endogenous circoviral elements in vertebrate genomes.
		<br>
		<i>Virus Research</i>
		<a href="https://www.sciencedirect.com/science/article/pii/S0168170217309048" target="_blank">[view]</a> 
		<br>
		<br>

		Dennis TPW, Flynn PJ, de Souza WM, Singer JB, Moreau CS, Wilson SJ, and RJ Gifford 
		<b>(2018)</b>
		<br>
		Insights into circovirus host range from the genomic fossil record.
		<br>
		<i>Journal of Virology</i>
		<a href="http://jvi.asm.org/content/92/16/e00145-18.full" target="_blank">[view]</a> 
		<br>
		<br>

		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>

		Zhu H, Dennis T, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		<a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		<br>
		<br>

		Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		<b>(2018)</b>
		<br>
		Nomenclature for endogenous retrovirus (ERV) loci.
		<br>
		<i>Retrovirology</i>
		<a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		<br>
		<br>

		Katzourakis A. and RJ. Gifford
		<b>(2010)</b>
		<br>
		Endogenous viral elements in animal genomes.
		<br>
		<i>PLoS Genetics</i>
		<a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>



	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/">Hepadnaviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



