<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-127420264-1"></script>
	<script>
	  window.dataLayer = window.dataLayer || [];
	  function gtag(){dataLayer.push(arguments);}
	  gtag('js', new Date());
	  gtag('config', 'UA-127420264-1');
	</script>


    <meta charset="UTF-8">
    <title>Hepadnaviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Hepadnaviridae-GLUE</h1>
      <h2 class="project-tagline">Paleovirology resources for hepadnaviruses.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="./project.html" class="btn">Viruses</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">

	   <h3>
		 <a id="How to use these resources" class="anchor" href="#PaleovirologyUses" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>A Hepadnaviridae-GLUE example for paleovirologists</strong>
	   </h3>

		<p>

		  This page provides a demonstration of Hepadnaviridae-GLUE being used to 
		  (i) organise the fossil record and; (ii) expand the fossil record to 
		  accomodate new data. 

		</p>


		<p>
		
		  Once it had been established which category a set of one or more sequences
		  belonged to, we processed it accordingly We used this resource to:
		  (i) implement comparative analyses that are highly reproducible, and;
		  (ii) make eHBV data available in a way that facilitates further investigation
		  in this area.
		  
		</p>

		<p>

          We included reference information the hepadnavirus family, including reference
          genomes, alignments, trees, and other auxilliary data. For reference genomes, 
          we used the library of reference genome sequences we constructed for DIGs (see above),
          linking these sequences with a record of their taxonomic and isolate-associated information 
          (Table 1). A set of (coding and non-coding) genome features was defined, and the 
          locations of these features on each reference sequence was recorded.
          
		  
		</p>

		<p>

          
          To incorporate EVEs the default data schema was extended to incorporate fields
          and tables representing the locations of genomic EVEs identified via DIGS,
          along with FASTA-formatted files representing the EVE sequences identified
          in the screen. We created a set of linked alignments that represented distinct
          taxonomic groups and levels within the family Hepadnaviridae. For the lower
          taxonomic levels (i.e. within and below genus level) we aligned complete coding
          sequences.
          
		  
		</p>

		<p>       
          For higher taxonomic levels we aligned the polymerase or RT genes. 
          We used GLUE to link these alignments, via a set of common reference sequences.
          The root alignment contains all reference sequences, whereas all children of the
          root inherit at least one reference from their immediate parent. Thus, all alignments are linked to one another via our chosen set of references.
          
		  
		</p>

	   <h3>
		 <a id="inheritFeatures" class="anchor" href="#PaleovirologyUses" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Inherit Features</strong>
	   </h3>

		<p>       

          We generated detailed annotations of HBV and DHBV master reference sequences and
          used GLUE’s ‘inherit feature locations’ function to infer their locations on
          other sequences.
          	  
		</p>

		<p>       

		  Record coverage relative to genus master reference genomes (see Methods), and
		  thereby infer the genomic structure of consensus eHBV elements  as well as that
		  of individual orthologs.
          	  
		</p>

		<p>       

		  makes it more challenging to infer the correct ORFs when there are indels
		  (i.e. in variable regions containing both overlapping ORFs and frameshifts
		  (see Figure 1) it may not be possible to infer which region of translated
		  sequence belongs to which gene.
          	  
		</p>

		<p>       


		  the margins for distinguishing between bona fide eHBV elements and distantly
		  related retroelements can at times be very thin.  HEART elements versus
		  cross-matching DIRS element
          
		  
		</p>


		<p>       

		  
		  
		  The designation of a new locus was based on (i) phylogenetic grouping if this
		  was clearly distinct from that of other eHBV elements, and 
		  (ii) the identification of unique flanking regions. 
		  Once a new eHBV locus had been identified, we used the DIGS framework to
		  identify any additional orthologs of this insertion within the set of sequences
		  previously obtained by screening.
		  
		</p>

		<p>       

		  Where appropriate, we also performed additional screens using the putative
		  polypeptide gene products (“puteins”) of eHBV sequences. In this way, we
		  systematically resolved the eHBV elements identified in our screen into
		  orthologous sets of sequences.



          	  
		</p>






	   <h3>
		 <a id="How to use these resources" class="anchor" href="#PaleovirologyUses" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Organising the fossil record</strong>
	   </h3>


		<p>

		  The presence of ancient orthologous eHBV insertions in certain vertebrate clades
		  (e.g. Neoaves, Serpentes, Testudines) quickly became apparent once the genome sequences
		  of multiple species were available for analysis - because these sequences were incorporated
		  into the germline prior to the radiation of major host lineages, they are present at
		  the same genomic locus in all species within those lineages.

		  The Hepadaviridae-GLUE resource forms an integral part of our framework for recovering the hepadnavirus fossil record. Firstly, it provides a stable source of ‘probe’ and ‘reference’ sequences for database-integrated genome screening (DIGS) [19].
		  Other points: degradation of EVEs.

		  Inconsistent start positions are found among hepadnaviruses sequences in GenBank.

		</p>





	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Hepadnaviridae-GLUE/">Hepadnaviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



